modules/meb_api/config/routes.rb

```diff
@@ -6,5 +6,6 @@
     get 'service_history', to: 'education_benefits#service_history'
     get 'eligibility', to: 'education_benefits#eligibility'
     get 'claim_status', to: 'education_benefits#claim_status'
+    post 'submit_claim', to: 'education_benefits#submit_claim'
```

Comment:
Is there updated swaqgger for this new endpoint?

Reply:
Actually, @LindseySaari , I was just talking to @bradbergeron-us and he tells me we have not added any swagger on this API as the services that this API will talk to are not yet finalized and it is not settled yet how the request/responses will look. At this time we are just adding mocks that allow the client to ping the service to allow us to make progress. Is it ok to add swagger on a later PR once we reach a final determination on the API?

Thanks

Cesar

Reply:
Yes definitely! Thanks for the thorough explanation!